dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/Startup.cs

```diff
@@ -10,11 +11,14 @@ public void ConfigureServices(IServiceCollection services)
         var dbHelper = new DbHelper(testConfiguration.Configuration.GetConnectionString("DefaultConnection") ??
             throw new Exception("Connection string DefaultConnection is missing."));
         var hostFixture = new HostFixture(testConfiguration, dbHelper);
-
         hostFixture.Initialize().GetAwaiter().GetResult();
 
+        var webhooksHostFixture = new WebHooksHostFixture(testConfiguration, dbHelper);
```

Comment:
This is going to end up doing a lot of initialization work more than once. I wonder whether not inheriting from `HostFixture` for these, as you said, would be better.

Reply:
Done